wlct/tournaments.py

```diff
@@ -2799,6 +2809,11 @@ def start(self):
             log("Tournament ID {} Name {} already started, and trying to start again!".format(self.name, self.id), LogLevel.critical)
             return
 
+        self.process_new_games_allowed = False
+
+        # start the round robin tournament
+        super(RoundRobinTournament, self).start()
```

Comment:
iirc, i moved this start() to the end of the function to prevent a race condition before. It has to do with the tournament being started, then new games being processed in start() later.